docs/makeown.rst

```diff
@@ -0,0 +1,12 @@
+makeown
+===========
+
+.. dfhack-tool::
+    :summary: Adds a unit to your fortress.
```

Comment:
this might be interpreted as "creates a unit". how about "Converts a unit to be a member of your fortress."?